etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json

```diff
@@ -1107,7 +1107,7 @@
                 "steppedLine": false,
                 "targets": [{
                     "refId": "A",
-                    "target": "groupByNode(tendrl.clusters.$cluster_id.volumes.$volume_name.nodes.$host_name.bricks.$brick_path.device.*.utilization.percent_used, 10, 'maxSeries')",
+                    "target": "groupByNode(tendrl.clusters.$cluster_id.volumes.$volume_name.nodes.$host_name.bricks.$brick_path.device.*.mount_utilization.percent_used, 10, 'maxSeries')",
```

Comment:
Woah, that was a quick fix!

Reply:
@r0h4n yes this is verified. @nthomas-redhat and I asked anmol that why the utilization is not coming. So he stated that there is a different parameter that which shows mount point utilizations so we need to show that.

That means that the utilization metric in brick is extra so we need to remove that later from the graphite.
linking: https://github.com/Tendrl/monitoring-integration/issues/228